samples/snippets/src/main/java/com/example/storage/object/PauseAndResumeAppendableObjectUpload.java

```diff
@@ -29,63 +29,78 @@
 import com.google.cloud.storage.StorageOptions;
 import com.google.common.io.ByteStreams;
 import java.io.IOException;
+import java.nio.ByteBuffer;
 import java.nio.channels.FileChannel;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Paths;
 import java.util.Locale;
 
-public class ResumeAppendableObjectUpload {
-  public static void resumeAppendableObjectUpload(
+public class PauseAndResumeAppendableObjectUpload {
+  public static void pauseAndResumeAppendableObjectUpload(
       String bucketName, String objectName, String filePath) throws Exception {
     // The ID of your GCS bucket
     // String bucketName = "your-unique-bucket-name";
 
-    // The ID of your GCS unfinalized appendable object
+    // The ID of your GCS object
     // String objectName = "your-object-name";
 
     // The path to the file to upload
     // String filePath = "path/to/your/file";
 
     try (Storage storage = StorageOptions.grpc().build().getService()) {
       BlobId blobId = BlobId.of(bucketName, objectName);
-      Blob existingBlob = storage.get(blobId);
-      BlobInfo blobInfoForTakeover = BlobInfo.newBuilder(existingBlob.getBlobId()).build();
+      BlobInfo blobInfo = BlobInfo.newBuilder(blobId).build();
+
+      // --- Step 1: Initial string write (PAUSE) ---
+      BlobAppendableUploadConfig initialConfig =
+          BlobAppendableUploadConfig.of().withCloseAction(CloseAction.CLOSE_WITHOUT_FINALIZING);
```

Comment:
`CLOSE_WITHOUT_FINALIZING` is the default close action. See https://cloud.google.com/java/docs/reference/google-cloud-storage/latest/com.google.cloud.storage.BlobAppendableUploadConfig#com_google_cloud_storage_BlobAppendableUploadConfig_withCloseAction_com_google_cloud_storage_BlobAppendableUploadConfig_CloseAction_ `Default:` for where this is documented